src/state/communication/messages/presentSlide.ts

```diff
@@ -21,11 +21,12 @@ import { Message } from '../connection';
 export const PRESENT_SLIDE_MESSAGE = 'net.nordeck.whiteboard.present_slide';
 
 export type PresentSlide = {
-  view: { slideId: string } | undefined;
+  view: { isEditMode: boolean; slideId: string } | undefined;
```

Comment:
`isEditMode` could also be optional. It would mean you don't need to change so many tests and it would be a backwards-compatible change. But it's fine to have it as required field.

Reply:
I am going to keep it required.